ca/src/main/scala/com/advancedtelematic/tuf/ca/db/SlickEncryptedColumn.scala

```diff
@@ -0,0 +1,75 @@
+package com.advancedtelematic.tuf.ca.db
+
+import java.util.Base64
+import javax.crypto.Cipher
+import javax.crypto.spec.PBEParameterSpec
+
+import io.circe.{Decoder, Encoder}
+
+import scala.reflect.ClassTag
+import slick.jdbc.MySQLProfile.api._
+import cats.syntax.either._
+import io.circe.parser
+import io.circe.syntax._
+import javax.crypto.SecretKeyFactory
+import javax.crypto.spec.PBEKeySpec
+
+import com.advancedtelematic.tuf.ca.Settings
+import org.bouncycastle.jce.provider.BouncyCastleProvider
+
+protected [db] class SlickCrypto(salt: Array[Byte], password: String) {
+  private lazy val pbeParameterSpec = new PBEParameterSpec(salt, 1000)
+  private val BC = BouncyCastleProvider.PROVIDER_NAME
+  private lazy val CIPHER_ALGORITHM = "AES/GCM/NoPadding"
```

Comment:
@heartsucker you probably want to take a look at this class and make sure I used the safe settings for key generation and encryption.